pygaggle/data/unicode.py

```diff
@@ -0,0 +1,22 @@
+import unicodedata
+import six
+
+
+def convert_to_unicode(text):
+  """Converts `text` to Unicode (if it's not already), assuming utf-8 input."""
+  if six.PY3:
+    if isinstance(text, str):
+      return text
+    elif isinstance(text, bytes):
+      return text.decode("utf-8", "ignore")
+    else:
+      raise ValueError("Unsupported string type: %s" % (type(text)))
+  elif six.PY2:
```

Comment:
Can we simply remove this and assume that python 3 is required?

Reply:
Yup! Good catch.